database/tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "strict": true,
     "typeRoots": ["node_modules/@types"]
   },
-  "include": ["src"],
+  "include": ["src", "20240722000002_remove_duplicate_users.ts", "20240722000002_remove_duplicate_users.ts"],
```

Comment:
Is this intentional?

Reply:
nope!

It likes to auto-add files when I move them around